### DISCUSSION TITLE
How to correctly set the compiler to ifort in the fpm.toml file.

### DISCUSSION THREAD

**Discussion by l1Ha** [fortran-lang/fpm#1069] [General]
How to correctly set the compiler to ifort in the fpm.toml file.

**Comment by perazz** [comment]
Hi, the compiler is not set in fpm.tom, but rather via command line interface or an environment variable. 

You can either run : 

```
fpm build --compiler ifort
```

or preset an environment variable `FPM_FC=ifort`.

See the online help in `fpm --help`.
